Jenkinsfile

```diff
@@ -0,0 +1,19 @@
+#!groovy
+
+node {
+ stage 'Checkout'
```

Comment:
Note that particular stage just does contain the `checkout scm` step... inspiration from `Jenkinsfile` examples like the one seen [here](https://github.com/jenkinsci/pipeline-examples/blob/master/jenkinsfile-examples/nodejs-build-test-deploy-docker-notify/Jenkinsfile)

That being said, definitely down to adjust wording/organization!